### DISCUSSION TITLE
Thảo luận về cấu trúc của dự án

### DISCUSSION THREAD

**Discussion by dinhgiaan** [dinhgiaandev/CodeGuru_BE#30] [Announcements]
- Cấu trúc dự án này đã hợp lý chưa?
- Cách clean code nhất để dễ maintenance
- Lựa chọn cổng PORT
- Các công nghệ sử dụng
- Các nơi lưu trữ dữ liệu

**Comment by dinhgiaan** [comment]
- Cổng PORT back-end nên lựa chọn 8000 hoặc 8001 hoặc 8888 tùy mỗi người.
- Nên chia tách các code ra từng phần nhỏ khác nhau để dễ maintenance (ví dụ khi đổi cổng PORT, cần dùng đến .env đặt cổng PORT...)

**Comment by dinhgiaan** [comment]
- Quản lý thông tin người dùng: nên dùng Mongodb, Redis...
- Quản lý hình ảnh: nên dùng Cloudianary

**Comment by ankieu123** [comment]
cổng be thì chọn 8888 đi, lỡ chọn 8000 hay 8011 nó bị lỗi trùng Port
